static/app/views/performance/landing/widgets/components/performanceWidget.tsx

```diff
@@ -189,6 +192,10 @@ const defaultGrid = {
   bottom: space(1),
 };
 
+const ErrorPanelWithMinHeight = styled(ErrorPanel)<{height: string}>`
+  min-height: ${p => p.height};
```

Comment:
```suggestion
  min-height: ${p => p.minHeight};
```

Reply:
Nit, but it might make sense to convert to px here and keep that value closer to CSS.